changelogs/fragments/9092-keycloak-clientscope-type-fix-check-mode.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

If this is a bugfix, this should be:
```suggestion
bugfixes:
```